src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import './App.scss';
+// eslint-disable-next-line import/no-named-as-default
+import Person from './components/Person/Person';
```

Comment:
The ESLint directive on line 3 is suppressing a warning about importing `Person` as a default export. Make sure that the `Person` component is indeed exported as a default export from './components/Person/Person'. If it's not, you may need to adjust the import statement accordingly.